edgenet/version.txt

```diff
@@ -1 +1 @@
-v0.10.0-rc2
+0.15.0-rc3
```

Comment:
we were somehow inconsistent with the `v` prefix and now no matter what we do we'll break some scripts :man_shrugging:

Reply:
yes, I based everything off mainnet which adds the v. so went with mainnet as the standard. happy to change it but I thought best not to break mainnet scripts